business_register/converter/pep.py

```diff
@@ -578,6 +580,8 @@ def save_or_update_peps_companies(self, peps_companies_data):
                     company.antac_id = company_antac_id
                     company_update_fields.append('antac_id')
             if not company:
+                if company_name_en is None and company_name:
```

Comment:
what if company company_name_en is empty string?
```suggestion
                if not company_name_en:
```